Examples/Examples/All Examples/SwiftUIExample.swift

```diff
@@ -0,0 +1,114 @@
+import UIKit
+import MapboxMaps
+import Turf
+import SwiftUI
+
+/// Example of how a SwiftUI map view can be used within a typical `ContentView`.
+public struct ContentView: View {
+
+    public var body: some View {
+        SwiftUIMapView(resourceOptions: ResourceOptions(accessToken: AccountManager.shared.accessToken!))
+            .styleURL(.dark)
+            .zoomLevel(14.0)
+            .centerCoordinate(CLLocationCoordinate2D(latitude: 20.6905, longitude: -88.2024))
+            .on(.mapLoadingFinished) { ( mapView ) in
+                let pointAnnotation = PointAnnotation(coordinate: mapView.centerCoordinate)
+                mapView.annotationManager.addAnnotation(pointAnnotation)
+            }
+    }
+}
+
+/// To render a `MapView` as a SwiftUI view, configure a new
+/// struct that implements the required `UIViewRepresentable` protocol.
+public struct SwiftUIMapView: UIViewRepresentable {
+
+    let resourceOptions: ResourceOptions
+    internal let observerConcrete: ObserverConcrete // Manages the observation of map events
+    internal var mapView: MapView
+
+    init(resourceOptions: ResourceOptions) {
+        self.resourceOptions = resourceOptions
+        self.mapView = MapView(with: .zero, resourceOptions: resourceOptions)
+        self.observerConcrete = ObserverConcrete(mapView: self.mapView)
+
+        // Subscribe to map events
+        let events = MapEvents.EventKind.allCases.map({ $0.rawValue })
```

Comment:
```suggestion
        let events = MapEvents.EventKind.allCases.map { $0.rawValue }
```

Code style - use trailing closure syntax or say `.map(\.rawValue)`